custom_components/tahoma/coordinator.py

```diff
@@ -105,6 +105,22 @@ async def _async_update_data(self) -> Dict[str, Device]:
                 event.new_state,
             )
 
+            if event.failure_type_code:
+                # Some event names are not mapped to an enum, it’s why this trick.
+                # Like CommandExecutionStateChangedEvent
+                try:
+                    raw_event_name = event.name.value
+                except AttributeError:
+                    raw_event_name = event.name
+
+                self.hass.bus.fire(
+                    f"Overkiz{raw_event_name}",
```

Comment:
Perhaps Overkiz_{}?

```suggestion
                    f"Overkiz_{raw_event_name}",
```

Reply:
Or maybe, make this more generic and add the specific event name to the payload. Or document the event names and how to catch them in specific scenario's.

Reply:
The name of the event is CameCase, it will be weird to prefix with `Overkiz_`